src/components/Navbar/Navbar.js

```diff
@@ -25,11 +61,13 @@ class NavbarUnlogged extends Component {
         <div className="collapse navbar-collapse" id="navbarSupportedContent">
           <form className="form-inline ml-auto my-2 my-lg-0">
             <input
+              id="searchBar"
               className="form-control mr-sm-2"
               type="search"
               placeholder="Szukaj"
               aria-label="Search"
               style={{ width: "400px" }}
+              onChange={event => this.setState({ phrase: event.target.value })}
```

Comment:
pieces of logic like this cannot be in component, introduce container for this one and move logic to it (including redux bindings)